src/utils/helper-functions.tsx

```diff
@@ -300,3 +300,6 @@ export const parseFileName = (
     fileExtension: fileArr[fileArr.length - 1]
   }
 }
+
+export const getInitials = (firstName: string, lastName: string): string =>
+  `${firstName[0]}${lastName[0]}`.toUpperCase()
```

Comment:
Return type of this function is redundant because it can be automatically infered as string

Reply:
What do you think about providing fallback value for `firstName` and `lastName` if they are not provided?

Reply:
Fixed